.github/workflows/mobilecoin-workflow-dev-deploy.yaml

```diff
@@ -331,11 +326,6 @@ jobs:
         image:
           org: ${{ inputs.docker_image_org }}
         fogIngest:
-          resources:
-            limits:
-              intel.com/sgx: 2000
-            requests:
-              intel.com/sgx: 2000
```

Comment:
Wouldn't we want these to be updated to the new resource definition?  We have lower limits in Dev than we do in Prod.